packages/core/admin/admin/src/components/GuidedTour/Context.tsx

```diff
@@ -127,7 +146,7 @@ const GuidedTourContext = ({
     enabled,
     completedActions: [],
   });
-  const migratedTourState = migrateTourSteps(storedTours);
+  const migratedTourState = migrateTours(storedTours);
```

Comment:
NOTE: migrateTourSteps is no longer necessary since we no longer store the length of the store in state / local storage. 

migrateTours is not _actually_ necessary but it future proofs the guided tour allowing tours to be added and removed between versions. I can remove it from the PR if you feel it should be its own PR

Reply:
not necessary at all? why not removing it then?

Reply:
Thanks! I'm not saying we should remove it, just trying to understand better